modules/n1ql/pages/n1ql-language-reference/metafun.adoc

```diff
@@ -368,6 +368,70 @@ The return value is a flattened list of array elements for use in an array index
 
 For examples, refer to xref:n1ql:n1ql-language-reference/indexing-arrays.adoc#examples[Array Indexing Examples].
 
+[[formalize,FORMALIZE()]]
+== FORMALIZE(`statement` [ `,query_context` ])
+
+=== Description
+
+Specifies all references from a query.
+
+Can be spelled as FORMALIZE or FORMALISE.
+
+=== Arguments
+
+statement:: A string containing the statement to formalize.
+
+query_context:: [ Optional ]
+A string query context value for the function to use when formalizing.
+
+=== Return Value
+
+Returns a query with all references fully specified.
+
+=== Examples
+
+[[formalize-ex,FORMALIZE() Example]]
+====
```

Comment:
As there's more than one example for FORMALIZE(), let's give the example a title

```suggestion
[[formalize-ex1,FORMALIZE() Example 1]]
.Formalize a query
====
```